it/controllers/ETMPControllerISpec.scala

```diff
@@ -92,6 +92,130 @@ class ETMPControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock {
       |}
       |""".stripMargin)
 
+  val etmpPayloadWithMultipleLSP_InSameCalenderMonth: JsValue = Json.parse(
```

Comment:
Could you rename this to:
```suggestion
  val etmpPayloadWithMultipleLSPInSameCalenderMonth: JsValue = Json.parse(
```